src/Client/Web/Bit.Client.Web.BlazorUI/Components/Pivot/BitPivot.razor.cs

```diff
@@ -0,0 +1,154 @@
+        public OverflowBehavior OverflowBehavior
+        {
+            get => _overflowBehavior;
+            set
+            {
+                _overflowBehavior = value;
+                ClassBuilder.Reset();
+            }
+        }
+
+        [Parameter]
+        public LinkFormat LinkFormat
+        {
+            get => _linkFormat;
+            set
+            {
+                _linkFormat = value;
+                ClassBuilder.Reset();
+            }
+        }
+
+        [Parameter]
+        public LinkSize LinkSize
+        {
+            get => _linkSize;
+            set
+            {
+                _linkSize = value;
+                ClassBuilder.Reset();
+            }
+        }
+
+        [Parameter]
+        public bool HeadersOnly { get; set; } = false;
+
+
+        [Parameter]
+        public EventCallback<BitPivotItem> OnLinkClick { get; set; }
+
+        [Parameter]
+        public string SelectedKey
+        {
+            get => selectedKey;
+            set
+            {
+                if (value == selectedKey) return;
+                selectedKey = value;
+                _ = SelectedKeyChanged.InvokeAsync(value);
+            }
+        }
+
+        [Parameter]
+        public EventCallback<string> SelectedKeyChanged { get; set; }
+
+        public IDictionary<string, BitPivotItem> Items = new Dictionary<string, BitPivotItem>();
+
+        protected override void OnInitialized()
+        {
+            selectedKey = selectedKey ?? DefaultSelectedKey;
+            base.OnInitialized();
+        }
+
+        protected override void RegisterComponentClasses()
+        {
+            ClassBuilder.Register(() => LinkSize == LinkSize.Large ? $"{RootElementClass}-large-{VisualClassRegistrar()}"
+                                      : LinkSize == LinkSize.Normal ? $"{RootElementClass}-normal-{VisualClassRegistrar()}"
+                                      : string.Empty);
+
+            ClassBuilder.Register(() => LinkFormat == LinkFormat.Links ? $"{RootElementClass}-links-{VisualClassRegistrar()}"
+                                      : LinkFormat == LinkFormat.Tabs ? $"{RootElementClass}-tabs-{VisualClassRegistrar()}"
+                                      : string.Empty);
+
+            ClassBuilder.Register(() => OverflowBehavior == OverflowBehavior.Menu ? $"{RootElementClass}-menu-{VisualClassRegistrar()}"
+                                      : OverflowBehavior == OverflowBehavior.Scroll ? $"{RootElementClass}-scroll-{VisualClassRegistrar()}"
+                                      : OverflowBehavior == OverflowBehavior.None ? $"{RootElementClass}-none-{VisualClassRegistrar()}"
+                                      : string.Empty);
+        }
+
+        internal async Task ItemClicked(KeyValuePair<string, BitPivotItem> item)
+        {
+            if (item.Value.IsEnabled is false) return;
+
+            await OnLinkClick.InvokeAsync(item.Value);
+
+            if (SelectedKeyChanged.HasDelegate)
```

Comment:
I think this is wrong, please review the code implemented in BitCheckBox

Reply:
fixed data binding